test-apps/display-test-app/src/frontend/EditingTools.ts

```diff
@@ -253,3 +253,68 @@ export class PlaceLineStringTool extends CreateElementTool {
       return this.exitTool();
   }
 }
+
+function orderIds(elementIds: string[]): OrderedId64Array {
+  const ids = new OrderedId64Array();
+  elementIds.forEach((id) => ids.insert(id));
+  return ids;
+}
+
+function compressIds(elementIds: string[]): CompressedId64Set {
+  const ids = orderIds(elementIds);
+  return CompressedId64Set.compressIds(ids);
+}
+
+async function startCommand(imodel: BriefcaseConnection): Promise<string> {
+  return EditTools.startCommand<string>({ commandId: editorBuiltInCmdIds.cmdBasicManipulation, iModelKey: imodel.key });
+}
+
+export async function transformElements(imodel: BriefcaseConnection, ids: string[], transform: Transform) {
+  await startCommand(imodel);
+  await basicManipulationIpc.transformPlacement(compressIds(ids), transform.toJSON());
+}
+
+/** This tool adds a seequent reality model to the viewport.
+ * @alpha
+ */
+export class MoveElementTool extends Tool {
+  public static override toolId = "MoveElement";
+  public static override get minArgs() { return 1; }
+  public static override get maxArgs() { return 4; }
+
+  /** This method runs the tool, adding a reality model to the viewport
+   * @param url the URL which points to the reality model tileset
+   */
+  public override async run(elementId: string, x: number, y: number, z: number): Promise<boolean> {
+
+    if (!IModelApp.viewManager.selectedView) {
+      return false;
+    }
+    const imodel = IModelApp.viewManager.selectedView.iModel;
+
+    if (imodel.isBriefcaseConnection()) {
+      await transformElements(imodel, [elementId], Transform.createTranslationXYZ(x, y, z));
+      await imodel.saveChanges();
+    }
+
+    return true;
+  }
+
+  /** Executes this tool's run method with args[0] containing the `url` argument.
+   * @see [[run]]
+   */
+  public override async parseAndRun(...args: string[]): Promise<boolean> {
+    let x = 0;
+    let y = 0;
+    let z = 1;
```

Comment:
1 is desired default for z, so `dta move element 0x123 1` will offset by (1, 0, 1)?

Reply:
changed default behavior